linode/instance/helpers.go

```diff
@@ -239,6 +243,7 @@ func updateInstanceConfigs(
 				"body":      configUpdateOpts,
 				"config_id": existingConfig.ID,
 			})
+			tflog.Debug(ctx, "zhiwei debug", map[string]any{"value": configUpdateOpts.Interfaces})
```

Comment:
It might make sense to convert this to an actual trace statement 👍

Reply:
Oops, this was a temp debug output, will remove